SWLOR.Game.Server/Feature/QuestDefinition/MonCalaQuestDefinition.cs

```diff
@@ -112,5 +113,23 @@ private void FishingGuildQuests()
                     Achievement.GiveAchievement(player, AchievementType.TheLegendaryRod);
                 });
         }
+
+        private void PartyRoomForPedro()
+        {
+            _builder.Create("partyroom_pedro", "Party Room for P3DR0")
+
+                .AddState()
+                .SetStateJournalText("P3DR0 wants a new place to party where they're not going to get kicked out of. Bring them five speakers, a jukebox and the schematics for a new cantina.")
+                .AddCollectItemObjective("structure_0272", 5)
+                .AddCollectItemObjective("structure_0005", 1)
+                .AddCollectItemObjective("structure_5004", 1)
+
+                .AddState()
+                .SetStateJournalText("Look's like P3DR0's going to be able to party. Make sure you talk to them!")
+
+                .AddGoldReward(1500)
+                .AddXPReward(1500)
+                .AddItemReward("recipe_fabdance1", 1);
```

Comment:
Resources of a few different tiers of many different types go into crafting the above items. You're more familiar with fabrication than me, but I get the feeling these rewards are a little small. Maybe buff the XP reward? Or have some high end food be rewarded as well?

Reply:
Have edited these to award 5000 Gold and 2500 XP to account for the materials used.